common/security/test/u_security_tls_test.c

```diff
@@ -362,7 +364,7 @@ U_PORT_TEST_FUNCTION("[securityTls]", "securityTlsSock")
         U_PORT_TEST_ASSERT(send(descriptor, gData, sizeof(gData) - 1) == sizeof(gData) - 1);
 
         U_TEST_PRINT_LINE("%d byte(s) sent via TCP @%d ms, now receiving...",
```

Comment:
%d -> %u why re we here using the uPortGetTickTimeMs directly lter we only report relative time. maybe this is not needed at all.

Reply:
For this and the similar ones below, I know it looks a bit peculiar, but I found it useful when staring at logs of misbehaving sockets connections to know when the target thought it was doing a thing (as opposed to the time-stamping that lines get when they are logged by the test system).  Since it is only test code my preference would to leave them there in case/(when :-)) this needs to be done in the future.

Reply:
I will correct the formatter/cast etc. though.